packages/desktopjs/src/Default/default.ts

```diff
@@ -240,6 +241,7 @@ export namespace Default {
             // Create a global windows object for tracking all windows and add the current global window for current
             if (this.globalWindow && !(DefaultContainer.windowsPropertyKey in this.globalWindow)) {
                 this.globalWindow[DefaultContainer.windowsPropertyKey] = { root: this.globalWindow };
+                this.globalWindow[DefaultContainer.windowNamePropertyKey] = this.globalWindow[DefaultContainer.windowUuidPropertyKey] = DefaultContainer.rootWindowUuid;
```

Comment:
This wasn't necessary to fix getMainWindow as root is already defined in line above, but makes win.id and win.name actually return defined values that can be used for comparing and getWindow* calls